README.md

```diff
@@ -1,18 +1,49 @@
+
+### Authorization (OpenID Connect)
+- For [Health](https://developer.va.gov/explore/health/docs/authorization) and [Veteran Verification](https://developer.va.gov/explore/verification/docs/authorization).  The VA API Platform uses the OpenID Connect standard to allow Veterans to authorize third-party applications to access data on their behalf. 
+
+### Benefits Claims
+- Use the [Claims](https://developer.va.gov/explore/benefits/docs/claims) (Auto-Establishment) API to submit a Veteran's benefits claim via EVSS (Electronic Veterans Self-Service), and to request the status of a Veteran's benefits claim.
+
+### Benefits Intake (fka Document Upload Service)
+- The [Benefits Intake](https://developer.va.gov/explore/benefits/docs/benefits) API allows authorized third-party systems used by Veteran Service Organizations and agencies to upload claim documents (scanned to PDF) directly to the Veterans Benefits Administration's (VBA) claims intake process.
+
+### Disability Rating
+- The [Disability Rating](https://developer.va.gov/explore/verification/docs/disability_rating) API allows third-parties to access the disability rating of a Veteran after receiving authorization to do so using an Open ID Connect flow.
+
+
+### Facilities
+- Use the [Facilities](https://developer.va.gov/explore/facilities/docs/facilities) API to find contact info, location, hours of operation, available services and other relevant information about a specific VA facility. 
+
+### Health (currently dev only)
+- Use the [Health APIs](https://developer.va.gov/explore/health/docs/argonaut) to allow Veterans to view their medical records, schedule an appointment, find a specialty facility, and share their information with caregivers and providers.
+
+### Service History
+- The [Service History](https://developer.va.gov/explore/verification/docs/service_history) API allows third-parties to access the service history of a Veteran after receiving authorization to do so using an Open ID Connect flow.
+
+### Veteran Confirmation
+- The [Veteran Confirmation](https://developer.va.gov/explore/verification/docs/veteran_confirmation) API allows third-parties to request confirmation from the VA of an individual's Veteran status after receiving authorization to do so using an Open ID Connect flow.
+
+\* APIs marked with an asterisk are currently for internal VA use only.
 
 ## Requesting Access
-We're still working to make these APIs generally available. In the meantime,
-please email `Alex.Yale-Loehr@va.gov` to request developer access. 
+If you have any questions after consulting the [API documentation](https://developer.va.gov/explore) and [applying](https://developer.va.gov/apply) for a developer key or OAuth credentials, please email api@va.gov for assistance.  You will receive a human reply within 24 hours.
```

Comment:
I think I understand what you are trying to do here (read the docs), but think it makes sense to be direct/simple here:

`If you have any questions or need further assistance, please email our team at api@va.gov. You will receive a human reply within 24 hours.`